contracts/NestedFactory.sol

```diff
@@ -496,14 +480,39 @@ contract NestedFactory is INestedFactory, ReentrancyGuard, OwnableProxyDelegatio
             unchecked {
                 uint256 underSpentAmount = _amountToSpend - amounts[1];
                 if (underSpentAmount != 0) {
-                    _safeTransferWithFees(IERC20(_inputToken), underSpentAmount, _msgSender(), _nftId);
+                    SafeERC20.safeTransfer(IERC20(_inputToken), _msgSender(), underSpentAmount);
                 }
             }
         } else {
-            _safeTransferWithFees(IERC20(_inputToken), _amountToSpend, _msgSender(), _nftId);
+            SafeERC20.safeTransfer(IERC20(_inputToken), _msgSender(), _amountToSpend);
         }
     }
 
+    /// @notice Transfer fees to the FeeSplitter
```

Comment:
The `@notice` should be replaced by a `@dev` because this function is private, it is more relevant for developers than for users.

Reply:
Done